### PR TITLE
ejabberd: added lager and elixir support

### PR DESCRIPTION
**Pull request by gongled**
Would be great to have `ejabberd` with lager (`--enable-lager`) and elixir (`--enable-elixir`) support. What do you think about this?

### CODE REVIEW

**Comment by andyone** [comment]
`--enable-elixir` not require installed elixir package for build?

**Comment by gongled** [comment]
It produces three files and I don't know why it happens. I suggest to remove `elixir` support temporary. I'm going to investigate this feature and prepare new pull request.

```
   /usr/bin/elixir
   /usr/bin/iex
   /usr/bin/mix
```

**Comment by andyone** [comment]
Ok. I agree.

File: ejabberd/ejabberd.spec [hunk 1]
```diff
@@ -162,6 +163,9 @@ getent passwd %{user_name} >/dev/null || %{__useradd} -d %{_sharedstatedir}/%{na
 ###############################################################################
 
 %changelog
+* Fri Apr 29 2016 Gleb Goncharov <g.goncharov@fun-box.ru> - 16.03-2
```
Comment (andyone) [inline ejabberd/ejabberd.spec:166]: You forgot bump release to this version

File: ejabberd/ejabberd.spec [hunk 2]
```diff
@@ -76,25 +76,25 @@ Provides:          %{name} = %{version}-%{release}
...
 
-ejabberd is taking great care of XMPP compliance, implementing most of the 
+ejabberd is taking great care of XMPP compliance, implementing most of the
 XMPP extensions published by the XMPP Standard Foundation.
 
-To innovate even further, the core development team is constantly working 
+To innovate even further, the core development team is constantly working
 with other open source communities to create bridges and elegant features.
 
-This ejabberd community site is a hub for all people that are interested in
```
Comment (andyone) [inline ejabberd/ejabberd.spec:97]: Don't remove spaces in description, it's not a problem, it's real spaces between words in description.

File: ejabberd/ejabberd.spec (hunk 1 above)
Comment (gongled) [inline ejabberd/ejabberd.spec:166]: Fixed (https://github.com/essentialkaos/kaos-repo/pull/62/commits/ef96c3a43add899b59d57a41ad22cc125319673e).

**Comment by andyone** [comment]
Looks good now.

**Pull request merged**